src/filters/UserFilters.jsx

```diff
@@ -69,8 +69,8 @@ export default class UserFilters extends React.Component {
   }
 
   setSpecificBreakdown(specificBreakdown) {
-    // console.log('setSpecificBreakdown', specificBreakdown.value);
-    this.props.dispatch(setSpecificBreakdown(specificBreakdown.value));
+    let newValue = specificBreakdown ? specificBreakdown.value : '';
```

Comment:
I think we can do this instead:

`let { value='' } = specificBreakdown;`

(just for the lulz)